lib/srv/alpnproxy/common/protocols.go

```diff
@@ -167,6 +169,8 @@ func ToALPNProtocol(dbProtocol string) (Protocol, error) {
 		return ProtocolOpenSearch, nil
 	case defaults.ProtocolDynamoDB:
 		return ProtocolDynamoDB, nil
+	case defaults.ProtocolClickHouse, defaults.ProtocolClickHouseHTTP:
+		return ProtocolClickhouse, nil
```

Comment:
Wonder if there is any benefit to creating a new ALPN protocol for each new database type, vs use a shared ALPN protocol something like `ProtocolDBNativeTLS`.